server/controllers/refer/referralsController.test.ts

```diff
@@ -200,13 +209,33 @@ describe('ReferralsController', () => {
       courseService.getCourse.mockResolvedValue(course)
       ;(CourseUtils.presentCourse as jest.Mock).mockReturnValue(coursePresenter)
 
-      const earliestCourseParticipation = courseParticipationFactory.build({ createdAt: '2022-01-01T12:00:00.000Z' })
-      const latestCourseParticipation = courseParticipationFactory.build({ createdAt: '2023-01-01T12:00:00.000Z' })
-      const summaryListOptions = 'summary list options'
+      const addedByUser = userFactory.build()
+
+      const earliestCourseParticipation = courseParticipationFactory.build({
+        addedBy: addedByUser.username,
+        createdAt: '2022-01-01T12:00:00.000Z',
+      })
+      const presentedEarliestCourseParticipation: CourseParticipationPresenter = {
+        ...earliestCourseParticipation,
+        addedByDisplayName: StringUtils.convertToTitleCase(addedByUser.name),
+      }
+      const latestCourseParticipation = courseParticipationFactory.build({
+        addedBy: addedByUser.username,
+        createdAt: '2023-01-01T12:00:00.000Z',
+      })
       courseService.getParticipationsByPerson.mockResolvedValue([
         latestCourseParticipation,
         earliestCourseParticipation,
       ])
+      const presentedLatestCourseParticipation: CourseParticipationPresenter = {
+        ...latestCourseParticipation,
+        addedByDisplayName: StringUtils.convertToTitleCase(addedByUser.name),
+      }
+      courseService.presentCourseParticipation
+        .mockResolvedValueOnce(presentedEarliestCourseParticipation)
+        .mockResolvedValueOnce(presentedLatestCourseParticipation)
```

Comment:
This relies on `presentCourseParticipation` being called only twice and in this specific order, right?

Probably fine given it's specific to this test, but using `jest-when` could avoid any confusion if we rearrange anything here (e.g. if we switch to reverse sorting). Something like (with the arguments flipped and the presenter naming convention update) `when(courseService.presentCourseParticipation).calledWith(token, earliestCourseParticipation).mockResolvedValue(earliestCourseParticipationPresenter)`

Same in the other controller